tests/functional/disaster-recovery/sc_arbiter/test_netsplit.py

```diff
@@ -85,23 +87,26 @@ def test_netsplit(
         duration,
         init_sanity,
         reset_conn_score,
+        setup_vms_standalone_pvc,
     ):
         """
         This test will test the netsplit scenarios when active-active CephFS and RBD workloads
```

Comment:
Please elaborate here what netsplit cases will be covered. E.g., ab-bc, b-c, ac-bc ..